### ISSUE TITLE
Unable to change the width of the widget rendered in the stepBuilder

### ISSUE THREAD

**Issue by cjouannet** [talamaska/onboarding_overlay#31]
Hello,

thanks to the version 3.0.0 of your package, we can now return a custom widget in the stepBuilder to render pretty much everything we want.
This is a great feature, however i noticed that the rendered widget takes the full width of the device and I can't change or constraints the width.

I tried returning, in the stepBuilder, a SizedBox with a width of half the one given by the **OnboardingStepRenderInfo** but it doesn't change the end result.

```dart
stepBuilder: (
    BuildContext context,
    OnboardingStepRenderInfo renderInfo,
) {
    return SizedBox(
        width: renderInfo.size.width / 2,
        child: Material(
            child: ...,
        ),
    );
},
```

I also tried to add some padding at the top of the widget in the stepBuilder method, but only worked by removing the **labelBoxDecoration**, and with that loosing the arrow, which isn't really what i want.

```dart
stepBuilder: (
    BuildContext context,
    OnboardingStepRenderInfo renderInfo,
) {
    return Padding(
        padding: const EdgeInsets.symmetric(horizontal: 20),
        child: Material(
            child: ...,
        ),
    );
},
```

So my question is, is there a way to adjust the width of the widget returned in the stepBuilder? Or does it not exist yet?

**Comment by talamaska** [comment]
I haven't tested that use-case. Will see what I can do and will be back with to you, maybe next week.

**Comment by talamaska** [comment]
fixed in [3.1.0-pre.2](https://pub.dev/packages/onboarding_overlay/versions/3.1.0-pre.2)
please test
